### PR TITLE
Past projects section

### LINKED ISSUE #32: Past Projects (2 weeks)
WEEK 1: DESKTOP
WEEK 2: MOBILE 

Note: 

1.  When developing on desktop, add the styling with consideration for what would be easiest to add media queries for on mobile. 
2. The desktop design applies to > tablet and mobile applies to < tablet

- [ ] Develop this section according to the design (including the view archive stuff) on desktop + mobile (responsive all sizes)
- [ ] Create a new component, PastProjectsSection in main-site and use it in the index.ts of main-site
- [ ] For the mobile version, use the Arrow component for the arrows and use the getLeftOrRight function in utils.ts to get the left or right item (see about section implementation as a reference). 
Screen Shot 2022-10-04 at 11 55 34 PM
Screen Shot 2022-10-04 at 11 55 23 PM
- [ ] Link the archive button to https://projects.hackbeanpot.com/

### PR DESCRIPTION
**Pull request by srazh**
Developed Past Projects section with responsiveness down to TabletLg ( greater than Tablet)

Fixes #32 

Changelist:

- created past projects section and corresponding style file
- used camel and fennec fox svg files
- project links should lead to github/devpost page

Tested:

- localhost8000 for different screen sizes

Screenshots & Screencasts:

https://user-images.githubusercontent.com/63278213/196312798-156ba984-0b2b-40ad-8752-28a8842704d9.mov

### CODE REVIEW

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: Between tablet and tabletLg, the proportion looks a bit off and the text is wrapping around the image. Can we make the image smaller here? After doing that if the proportion still looks off u could also try increasing the width of the dark brown background thing its on 

Screen Shot 2022-10-18 at 3 36 52 PM

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: change to right: 0; you don't need to specify the measurement if it's 0!

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: same here

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em for everything !

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em for everything!

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [reply main-site/components/past-projects-section/PastProjectsSection.styles.ts]: also if possible use em here - if that's being weird for images just do px then

File: main-site/components/past-projects-section/PastProjectsSection.tsx
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.tsx]: Clicking this link takes me here
Screen Shot 2022-10-18 at 3 41 58 PM

Link this instead https://devpost.com/software/oh-duck

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
OMG SARAH THIS LOOKS SOOOOOO GOOOD!!!!! Wow you're such a fast learner!!!! Overall looks AMAZING - just a few small comments!

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jhylisa666) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts:135]: I think in the actual design, the styled fox is smaller so that it doesn't end up being right next to the texts

Comment (jhylisa666) [reply main-site/components/past-projects-section/PastProjectsSection.styles.ts:135]: or i guess you can also do <br /> new line after the text like the design so it's not right next to the fox ^^

**Review by jhylisa666 — CHANGES_REQUESTED** [review]
Also nitpick but maybe put a bit more padding between the bottom of the Oh Duck image and the brown container?(like the amount of padding at the top) :))))
but GREAT JOB KWEEEEEEN!!!!!!!!

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (srazh) [reply main-site/components/past-projects-section/PastProjectsSection.styles.ts:135]: made fox smaller

**Review by jzhzhang66 — APPROVED** [review]

**Review by jhylisa666 — APPROVED** [review]
LGTMMM

**Comment by srazh** [comment]
Developed Past Projects section with responsiveness down to mobile (< tablet)

Fixes https://github.com/HackBeanpot/mono-repo/issues/32

Changelist:
 - responsiveness for mobile
 - carousel section

Tested:

localhost8000 for different screen sizes
Screenshots & Screencasts:


https://user-images.githubusercontent.com/63278213/197685482-446bf7c2-12cd-4d5d-ae69-53ea5e7b91ab.mov

**Review by jhylisa666 — CHANGES_REQUESTED** [review]
GREAT JOBBB SARAH!!!!!
1. I think for non desktop there's still a giraffe at the top. 
2. Also on the design i think the coloring for "inky the black hole pet" is black, instead of cyan. And the paragraphs are brown and black. 
3. Think the arrows should be on the same line as the title as opposed to the by-line. If you display flex and put the arrow and the title in a div think that might fix it.

**Comment by jzhzhang66** [comment]
+ 1 to everything @jhylisa666 said!! And also, for this "interested in seeign more...' text - can we adding some padding left / right? At some screensizes its touching the edge of the screen

Screen Shot 2022-10-25 at 1 25 54 PM

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
See prev comment about text touching the edge of screens

**Comment by srazh** [comment]
fixed requested changes except Lisa's #3, I actually prefer the arrows to be the position they are in right now (I think it's better design compared to the figma design lol)

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: Remove this div if there's nothing in it / not being used

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: Use em

File: main-site/components/past-projects-section/PastProjectsSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/past-projects-section/PastProjectsSection.styles.ts]: use em

**Review by jzhzhang66 — APPROVED** [review]
Approving with a few comments if you could please change those before you merge! 

I am forgetting if you were running into issues with using em for images? If you're able to use em for width + height let's do that

**Review by jhylisa666 — APPROVED** [review]
LGTMMM

**Pull request merged**
